diff --git a/src/WallpaperEngine/WebBrowser/CEF/RenderHandler.cpp b/src/WallpaperEngine/WebBrowser/CEF/RenderHandler.cpp
index fc31baad..72e97845 100644
--- a/src/WallpaperEngine/WebBrowser/CEF/RenderHandler.cpp
+++ b/src/WallpaperEngine/WebBrowser/CEF/RenderHandler.cpp
@@ -27 +27 @@ int RenderHandler::getHeight () const { return this->m_webdata->getHeight (); }
-GLuint RenderHandler::texture () const { return this->m_webdata->getWallpaperFramebuffer (); }
\ No newline at end of file
+GLuint RenderHandler::texture () const { return this->m_webdata->getWallpaperTexture (); }
\ No newline at end of file
